### PR TITLE
Christ. Updates rover_logs to use a sequence offset.

### PR DESCRIPTION
**Pull request by mookerji**
/cc @henryhallam

### CODE REVIEW

File: gnss_analysis/tools/records2table.py [hunk 1]
```diff
@@ -101,6 +102,7 @@ def _process_obs(self, host_offset, host_time, msg):
              'cn0': o.cn0, 'lock': o.lock}
         v.update({'host_offset': host_offset, 'host_time': host_time})
         if time in t:
+          assert not t[time].get(prn, None)
```
Comment (henryhallam) [inline gnss_analysis/tools/records2table.py]: what does this do?

File: gnss_analysis/tools/records2table.py [hunk 2]
```diff
@@ -190,9 +192,18 @@ def _process_iar(self, host_offset, host_time, msg):
   def _process_log(self, host_offset, host_time, msg):
     if type(msg) is lg.MsgPrint:
       m = exclude_fields(msg)
-      m['host_offset'] = host_offset
+      k = host_offset
+      # MsgPrints sometimes coincide in time, so add a deterministic
```
Comment (henryhallam) [inline gnss_analysis/tools/records2table.py]: conceivably any messages can arrive within 1ms, not just prints

Comment (mookerji) [inline gnss_analysis/tools/records2table.py]: Indeed.

File: gnss_analysis/tools/records2table.py (hunk 1 above)
Comment (mookerji) [inline gnss_analysis/tools/records2table.py]: asserts if MsgObs has the same issue. I should probably just unify the host timing handling across all the message types..

File: gnss_analysis/tools/records2table.py (hunk 2 above)
Comment (mookerji) [inline gnss_analysis/tools/records2table.py]: Better safe than sorry, I suppose.

**Pull request merged**